Formula/caddy.rb

```diff
@@ -68,17 +63,17 @@ class Caddy < Formula
 
   go_resource "github.com/lucas-clemente/quic-go" do
     url "https://github.com/lucas-clemente/quic-go.git",
-        :revision => "5f25ffc795f3f7c60586123e550c0d01b9f6cfb4"
+        :revision => "8489c94f4d3aaa5feaa284a16376cfb8ea7d2c2c"
   end
 
   go_resource "github.com/lucas-clemente/quic-go-certificates" do
     url "https://github.com/lucas-clemente/quic-go-certificates.git",
         :revision => "d2f86524cced5186554df90d92529757d22c1cb6"
   end
 
-  go_resource "github.com/mholt/archiver" do
-    url "https://github.com/mholt/archiver.git",
-        :revision => "c55f48c22890807c9698370fe5e08eca3704834b"
+  go_resource "github.com/mholt/caddy" do
```

Comment:
No. I think gdm included it because I was running gdm from `github.com/mholt/caddy/caddy`, and there are other components `github.com/mholt/caddy/caddyfile`, `github.com/mholt/caddy/caddyhttp`, etc..